great_expectations_cloud/agent/agent.py

```diff
@@ -327,9 +426,9 @@ def _handle_event_as_thread_exit(
                     extra={
                         "event_type": event_context.event.type,
                         "correlation_id": event_context.correlation_id,
-                        "job_duration": result.job_duration.total_seconds()
-                        if result.job_duration
-                        else None,
+                        "job_duration": (
+                            result.job_duration.total_seconds() if result.job_duration else None
```

Comment:
This seems to be out of scope of your changes, but a `job_duration` of 0 seconds would result in `None`

```suggestion
                            result.job_duration.total_seconds() if result.job_duration else None
```

Reply:
out of scope